policies/tool_policies/monitor_ec2.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This file should be called monitor_resources.json or something similar since it includes other resources than EC2